Doc/source/conf.py

```diff
@@ -24,9 +24,10 @@
 # Add build directory to the path, preferring version-specific.
 buildbase = os.path.abspath(os.path.join(
     os.path.dirname(__file__), '..', '..', 'build'))
-for pth in ('lib.{0}-{1}.{2}'.format(sysconfig.get_platform(),
+for pth in ('lib', # Prepending, so add low-priority paths first.
+            'lib.{0}-{1}.{2}'.format(sysconfig.get_platform(),
                                      *sys.version_info[:2]),
-            'lib'):
+):
```

Comment:
The lack of indent here seems odd. I'd have thought that there should at least be *some* indent. (my linter agrees).
I'd suggest that the sad face should be indented so that it's aligned with the hanging indent on line 28.

Reply:
yep